README.md

```diff
@@ -24,8 +24,55 @@
 
 Several quick start options are available:
 
-- Clone the repo: `git clone https://github.com/Orange-OpenSource/ods-charts.git`
-- Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
+- npm installation:
+  - Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
+  - Use in your js files : `import * as ODSCharts from 'ods-charts';`
+- Add javascript tag in your HTML page:
+
+```html
+<script src=" https://cdn.jsdelivr.net/npm/echarts@latest/dist/echarts.min.js "></script>
```

Comment:
Are we sure we want to recommend this approach in terms of eco-design? Users should probably only load what they need, and not the entire Apache ECharts library.

Reply:
Maybe not recommendation but it's still a way to achieve it

Reply:
I can remove reference to cdn.jsdeliver.net and replace it by a [path to the echarts.min.js]

and keep the npm install for both methods

Reply:
[jsDelivr not recomended](https://github.com/Orange-OpenSource/ods-charts/pull/275/commits/4271b0aa85ed5f75f9aaff35d3c75d4e40ea10b4)

Reply:
Yes I agree with you (even more since we are still using it in the doc :eyes:)

Reply:
We changed a little bit the wording with @louismaximepiton to explain it